sentry-compose/build.gradle.kts

```diff
@@ -0,0 +1,121 @@
+import io.gitlab.arturbosch.detekt.Detekt
+import io.gitlab.arturbosch.detekt.extensions.DetektExtension
+
+plugins {
+    kotlin("multiplatform")
+    id("com.android.library")
+    id("org.jetbrains.compose")
+    jacoco
+    id(Config.QualityPlugins.gradleVersions)
+    id(Config.QualityPlugins.detektPlugin)
+    `maven-publish`
```

Comment:
isn't `com.vanniktech:gradle-maven-publish-plugin` applied to every module already?

Reply:
The problem was that it wasn't publishing correct artifacts when I tested it with mavenLocal. I think it still publishes correctly to mavenCentral though, I've seen someone is using `maven-publish` exclusively for mavenLocal, I guess I will leave a comment there

Reply:
I've just compared the `pom` file with the okhttp one and I'd say it looks correct. I think the plugin is smart enough to pick up values like developer name, license, etc.
```
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <!-- This module was also published with a richer model, Gradle metadata,  -->
  <!-- which should be used instead. Do not delete the following line which  -->
  <!-- is to indicate to Gradle or any Gradle module metadata file consumer  -->
  <!-- that they should prefer consuming it instead. -->
  <!-- do_not_remove: published-with-gradle-metadata -->
  <modelVersion>4.0.0</modelVersion>
  <groupId>io.sentry</groupId>
  <artifactId>sentry-compose-android</artifactId>
  <version>6.1.1</version>
  <packaging>aar</packaging>
  <name>Sentry SDK</name>
  <description>SDK for sentry.io</description>
  <url>https://github.com/getsentry/sentry-java</url>
  <licenses>
    <license>
      <name>MIT</name>
      <url>http://www.opensource.org/licenses/mit-license.php</url>
    </license>
  </licenses>
  <developers>
    <developer>
      <id>getsentry</id>
      <name>Sentry Team and Contributors</name>
      <url>https://github.com/getsentry/</url>
    </developer>
  </developers>
  <scm>
    <connection>scm:git:git://github.com/getsentry/sentry-java.git</connection>
    <developerConnection>scm:git:ssh://git@github.com/getsentry/sentry-java.git</developerConnection>
    <url>https://github.com/getsentry/sentry-java</url>
  </scm>
  <dependencies>
    <dependency>
      <groupId>androidx.compose.runtime</groupId>
      <artifactId>runtime</artifactId>
      <version>1.1.0</version>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>androidx.compose.ui</groupId>
      <artifactId>ui</artifactId>
      <version>1.1.0</version>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>io.sentry</groupId>
      <artifactId>sentry</artifactId>
      <version>6.1.1</version>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>io.sentry</groupId>
      <artifactId>sentry-android-navigation</artifactId>
      <version>6.1.1</version>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>androidx.navigation</groupId>
      <artifactId>navigation-compose</artifactId>
      <version>2.4.2</version>
      <scope>compile</scope>
    </dependency>
    <dependency>
      <groupId>org.jetbrains.kotlin</groupId>
      <artifactId>kotlin-stdlib</artifactId>
      <version>1.6.10</version>
      <scope>runtime</scope>
      <exclusions>
        <exclusion>
          <artifactId>kotlin-stdlib-common</artifactId>
          <groupId>org.jetbrains.kotlin</groupId>
        </exclusion>
      </exclusions>
    </dependency>
    <dependency>
      <groupId>androidx.lifecycle</groupId>
      <artifactId>lifecycle-common-java8</artifactId>
      <version>2.2.0</version>
      <scope>runtime</scope>
      <exclusions>
        <exclusion>
          <artifactId>kotlin-stdlib-common</artifactId>
          <groupId>org.jetbrains.kotlin</groupId>
        </exclusion>
      </exclusions>
    </dependency>
  </dependencies>
</project>

```

Reply:
Cool, `gradle-maven-publish-plugin` is doing its work as well then.